helper-cli/src/main/kotlin/commands/CreateFlatAnalyzerResultCommand.kt

```diff
@@ -0,0 +1,181 @@
+import org.ossreviewtoolkit.utils.common.expandTilde
+
+internal class CreateFlatAnalyzerResultCommand : CliktCommand(
+    "A command which turns a simple definition file into an analyzer result."
+) {
+    private val simpleDefinitionFile by option(
+        "--simple-definition-file", "-i",
+        help = "The simple definition file to read the project definition from."
+    ).convert { it.expandTilde() }
+        .file(mustExist = false, canBeFile = true, canBeDir = false, mustBeWritable = false, mustBeReadable = false)
+        .convert { it.absoluteFile.normalize() }
+        .required()
+
+    private val ortFile by option(
+        "--ort-file", "-o",
+        help = "The ORT file to write the generated synthetic analyzer result to."
+    ).convert { it.expandTilde() }
+        .file(mustExist = false, canBeFile = true, canBeDir = false, mustBeWritable = false, mustBeReadable = false)
+        .convert { it.absoluteFile.normalize() }
+        .required()
+
+    override fun run() {
+        val definitionFile = FileFormat.forFile(simpleDefinitionFile).mapper.readValue<SimpleDefinitionFile>(
+            simpleDefinitionFile
+        )
+
+        val projectVcs = VcsInfo(
+            type = definitionFile.vcsType ?: VcsType.UNKNOWN,
+            url = definitionFile.vcsUrl.orEmpty(),
+            revision = definitionFile.vcsRevision.orEmpty(),
+            path = definitionFile.vcsPath.orEmpty()
+        )
+
+        val projectName = definitionFile.name?.takeUnless { it.isBlank() } ?: DEFAULT_PROJECT_NAME
+
+        val project = Project.EMPTY.copy(
+            id = Identifier("Unmanaged::$projectName:"),
+            vcs = projectVcs,
+            vcsProcessed = projectVcs.normalize(),
+            scopeDependencies = setOfNotNull(
+                definitionFile.dependencies.filterNot { it.isExcluded }.toScope(MAIN_SCOPE_NAME),
+                definitionFile.dependencies.filter { it.isExcluded }.toScope(EXCLUDED_SCOPE_NAME)
+            )
+        )
+
+        val ortResult = OrtResult(
+            analyzer = AnalyzerRun.EMPTY.copy(
+                result = AnalyzerResult(
+                    projects = setOf(project),
+                    packages = definitionFile.dependencies.mapTo(mutableSetOf()) { it.toPackage() }
+                )
+            ),
+            repository = Repository(
+                vcs = projectVcs.normalize(),
+                config = RepositoryConfiguration(
+                    excludes = Excludes(
+                        scopes = listOf(
+                            ScopeExclude(EXCLUDED_SCOPE_NAME, ScopeExcludeReason.DEV_DEPENDENCY_OF)
+                        )
+                    )
+                )
+            )
+        )
+
+        writeOrtResult(ortResult, ortFile)
+    }
+}
+
+private const val DEFAULT_PROJECT_NAME = "unknown"
+private const val EXCLUDED_SCOPE_NAME = "excluded"
+private const val MAIN_SCOPE_NAME = "main"
+
+private data class SimpleDefinitionFile(
```

Comment:
How about annotating the class with `@JsonInclude(JsonInclude.Include.NON_DEFAULT)` instead of the individual annotations below?